### ISSUE TITLE
A session is either terminated or not started

### ISSUE THREAD

**Issue by yukinotech** [appium/appium#13734]
## The problem
 [qusetion](https://github.com/webdriverio/webdriverio/issues/4844)

come from `wdio`,is it a appium bug or wdio bug?

**Comment by yukinotech** [comment]
it seems I have solved this problem .
I change the src in `node_modules`,then it work.
I use `yarn add appium` to install,version is `1.15.1`,
then in folder `node_modules/appium/build/lib/appium.js`
I found the functiion `attachUnexpectedShutdownHandler`, then i delete it .
```
// this.attachUnexpectedShutdownHandler(d, innerSessionId);

 _logger.default.info(`New ${InnerDriver.name} session created successfully, session ` + `${innerSessionId} added to master session list`);
```
now it can work ,and does not delete session, is this a bug? if it is ,I would like to contribute a pr.
